cni/network/stateless/main.go

```diff
@@ -0,0 +1,161 @@
+// Copyright 2017 Microsoft. All rights reserved.
+// MIT License
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/Azure/azure-container-networking/cni"
+	zapLog "github.com/Azure/azure-container-networking/cni/log"
+	"github.com/Azure/azure-container-networking/cni/network"
+	"github.com/Azure/azure-container-networking/common"
+	"github.com/Azure/azure-container-networking/log"
+	"github.com/Azure/azure-container-networking/nns"
+	"github.com/Azure/azure-container-networking/platform"
+	"github.com/Azure/azure-container-networking/telemetry"
+	"github.com/pkg/errors"
+	"go.uber.org/zap"
+)
+
+var logger = zapLog.CNILogger.With(zap.String("component", "cni-main"))
+
+const (
+	hostNetAgentURL = "http://168.63.129.16/machine/plugins?comp=netagent&type=cnireport"
```

Comment:
some of these look common across regular cni

Reply:
Will be corrected on next commit